source/client-side-encryption/tests/unified/QE-Text-cleanupStructuredEncryptionData.yml

```diff
@@ -0,0 +1,128 @@
+description: QE-Text-cleanupStructuredEncryptionData
+schemaVersion: "1.23"
+runOnRequirements:
+  # Requires libmongocrypt 1.15.0 for SPM-4158.
```

Comment:
What would you think about making libmongocrypt version a runOnRequirement?  This would be useful in Node and maybe other drivers who have bindings separate from their driver code.  We actually do run our latest driver test suite against an older version of our bindings, so this would be necessary in Node (if it isn't codified into the UTR, we'll just have to skip it manually).

Reply:
We probably want to adopt these changes in https://github.com/mongodb/specifications/pull/1784.  I added a comment to that PR about these UTR changes.